FMDB.podspec

```diff
@@ -26,24 +31,28 @@ Pod::Spec.new do |s|
   s.subspec 'standalone' do |ss|
     ss.xcconfig = { 'OTHER_CFLAGS' => '$(inherited) -DFMDB_SQLITE_STANDALONE' }
     ss.dependency 'sqlite3'
-    ss.source_files = 'src/fmdb/FM*.{h,m}'
-    ss.exclude_files = 'src/fmdb.m'
-  end
-
-  # build with FTS support and custom FTS tokenizer source files
+    ss.dependency 'FMDB/common'
+	end
+	
+  # build the latest stable version of sqlite3 
```

Comment:
I'd tweak this comment to something like: 
`# Use the standalone version of sqlite3/fts and include custom FTS Tokenizer code`